gns3server/modules/qemu/__init__.py

```diff
@@ -39,6 +40,27 @@ class Qemu(BaseManager):
     _VM_CLASS = QemuVM
 
     @staticmethod
+    def get_kvm_archs():
+        """
+        Gets a list of architectures for which KVM is available on this server.
+
+        :returns: List of architectures for which KVM is available on this server.
+        """
+        kvm = []
+        x86_64_aliases = ["x86_64", "x86-64", "x64", "AMD64", "amd64", "Intel 64", "EM64T"]
+        i386_aliases = ["i386", "x86", "x32"]
+        if sys.platform.startswith("linux") and subprocess.call("kvm-ok") == 0:
```

Comment:
We will need to use an async subprocess call otherwise the server will freeze until kvm answer